services/viva-ms/lambdas/submitApplication.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-console */
+import AWS from 'aws-sdk';
+import to from 'await-to-js';
+import Hashids from 'hashids';
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+
+// import config from '../../../config';
```

Comment:
Remove unused import

Reply:
The use of config is implemented in a later commit